charts/portworx/v2.8.1/templates/portworx-csi.yaml

```diff
@@ -65,32 +117,68 @@ spec:
       containers:
         - name: csi-external-provisioner
           imagePullPolicy: Always
-          image: "{{ template "px.getcsiProvisioner" . }}/csi-provisioner:v0.2.0"
+          image: "docker.io/openstorage/csi-provisioner:v1.6.1-1"
+          args:
+            - "--v=3"
+            - "--provisioner=pxd.portworx.com"
+            - "--csi-address=$(ADDRESS)"
+            - "--enable-leader-election"
+            - "--leader-election-type=leases"
+          env:
+            - name: ADDRESS
+              value: /csi/csi.sock
+          securityContext:
+            privileged: true
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /csi
+        - name: csi-snapshotter
+          imagePullPolicy: Always
+          image: k8s.gcr.io/sig-storage/csi-snapshotter:v4.0.0
```

Comment:
The latest minor version for snapshotter is k8s.gcr.io/sig-storage/snapshot-controller:v4.2.1 - the following images can be used:
https://github.com/portworx/px-installer/blob/master/px-spec-websvc/csi.go#L244-L251